pkgs/desktops/arcan/pipeworld/default.nix

```diff
@@ -5,13 +5,13 @@
 
 stdenv.mkDerivation rec {
   pname = "pipeworld";
-  version = "0.pre+unstable=2021-08-01";
+  version = "0.pre+date=2021-11-26";
```

Comment:
Can we please wait until the rfc is done to change the version formats?

Reply:
For all the relevant things, there is no such standard for versions. So, there is little to no problem modifying them here.

Also, I am modifying only packages with new releases. Nothing will be lost.